opencog/guile/SchemeSmobTV.cc

```diff
@@ -460,5 +460,32 @@ SCM SchemeSmob::ss_tv_get_value (SCM s)
 	return SCM_EOL;
 }
 
+/**
+ * Return the truth value mean
+ */
+SCM SchemeSmob::ss_tv_get_mean(SCM s)
+{
+	TruthValue *tv = verify_tv(s, "cog-tv-mean");
+	return scm_from_double(tv->getMean());
+}
```

Comment:
I confirm, the scheme helpers tv-mean, tv-conf and tv-count return #t when given an IndefiniteTV.

These helpers are only used in the OpenPsi, and in all instances it would shorten the code a bit as cog-tv-mean, etc, are called directly on the atom.

So I suggest to remove the helpers. I assume you agree and I'll proceed.